config.go

```diff
@@ -82,6 +83,23 @@ func (c *Config) InitDefault(log *zap.Logger) {
 	case httpClient:
 	default:
 		c.Client = httpClient
+		// As per https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol
+		exporterEnv := "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"
```

Comment:
It is better to move these values into constants

Reply:
you want separate constants for self-descriptive environment variables? i.e. `const otelExporterOtlpTracesProtocolEnv = "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL"`? Isn't that a bit redundant?

Reply:
Hard to say, let's leave them at them moment as regular variables. But if we need to add more, we'll translate them into constants.